GrandCentralBoard/Widgets/Harvest/HarvestSource.swift

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
I doubt we'll change it in the nearest future, and I prefer to stick to the simpler name whenever possible.